ocs_ci/deployment/deployment.py

```diff
@@ -904,6 +905,22 @@ def create_catalog_source(image=None, ignore_upgrade=False):
     logger.info("Adding CatalogSource")
     if not image:
         image = config.DEPLOYMENT.get('ocs_registry_image', '')
+    if config.DEPLOYMENT.get('stage_rh_osbs'):
+        image = config.DEPLOYMENT.get(
+            "stage_index_image", constants.OSBS_BOUNDLE_IMAGE
+        )
+        osbs_image_tag = config.DEPLOYMENT.get(
+            "stage_index_image_tag", f"v{get_ocp_version()}"
+        )
+        image += f":{osbs_image_tag}"
```

Comment:
Didn't see the usage of "image" in if block and we are overriding the image in https://github.com/red-hat-storage/ocs-ci/pull/3039/files#diff-153ccf5916ca230718b5455c349afbcdR929

Reply:
It's used.

```
image_and_tag = image.split(':')
```

So if we have the image there then it's continue with tag passed and it's not overwritten.

I've tested it and see that from the job I've used for testing the proper image:tag was used as intended.

Reply:
ahh, missed that